shell/platform/android/test/io/flutter/plugin/editing/InputConnectionAdaptorTest.java

```diff
@@ -1104,16 +1106,25 @@ public void testSendKeyEvent_sendHardwareKeyEvents() {
 
   @Test
   public void testSendKeyEvent_delKeyNotConsumed() {
-    ListenableEditingState editable = sampleEditable(5, 5);
+    ListenableEditingState editable = sampleEditable(5, 5, "01234");
     InputConnectionAdaptor adaptor = sampleInputConnectionAdaptor(editable);
 
-    KeyEvent downKeyDown = new KeyEvent(KeyEvent.ACTION_DOWN, KeyEvent.KEYCODE_DEL);
+    KeyEvent downKeyUP = new KeyEvent(KeyEvent.ACTION_UP, KeyEvent.KEYCODE_DEL);
```

Comment:
This test was added by @gspencergoog  in https://github.com/flutter/engine/pull/22626. 
Is this change correct, Greg?